# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_yodlee
# 
# Translators:
# Martin Trigaux, 2020
# oihane <oihanecruce@gmail.com>, 2020
# mgalarza005 <mikelgalarza99@gmail.com>, 2020
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~12.5+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-08-26 08:04+0000\n"
"PO-Revision-Date: 2019-08-26 09:34+0000\n"
"Last-Translator: mgalarza005 <mikelgalarza99@gmail.com>, 2020\n"
"Language-Team: Basque (https://www.transifex.com/odoo/teams/41243/eu/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: eu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "(Diagnostic: %r for URL %s)"
msgstr "Diagnostikoa "

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Account"
msgstr "Kontua"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Already Registered (436):Your Auto-Registration could not be "
"completed because the site reports that your account is already registered."
"  Please log in to the site to confirm and then complete the site addition "
"process with the correct login information."
msgstr ""
"Erregistratuta dagoen kontua (436):Zure erregistro automatikoa ezin da "
"osatu, guneak zure kontua erregistratuta dagoela jakinarazten baitu. "
"Mesedez, hasi saioa gunean berresteko, eta, ondoren, osatu gunea gehitzeko "
"prozesua login-informazio zuzenarekin."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Locked (407): We could not update your account because it appears "
"your account has been locked. This usually results from too many "
"unsuccessful login attempts in a short period of time. Please visit the site"
" or contact its customer support to resolve this issue.  Once done, please "
"update your account credentials in case they are changed."
msgstr ""
"Kontua blokeatuta (407): Ezin dugu zure kontua eguneratu zure kontua "
"blokeatu dela dirudielako. Hori, normalean, denbora laburrean arrakasta izan"
" ez duten login-saio askorengatik gertatzen da. Mesedez, joan gunera edo "
"jarri harremanetan bezeroarentzako arreta-zerbitzuarekin arazo hori "
"konpontzeko. Hori egin ondoren, eguneratu zure kontuko kredentzialak, "
"aldatzen badira."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Not Found(408): We're sorry, we couldn't find any accounts for you "
"at the site. Please log in at the site and confirm that your account is set "
"up, then try again."
msgstr ""
"Ez da kontua aurkitu(408): Sentitzen dugu, ezin izan genuen zure konturik "
"aurkitu. Mesedez, hasi saioa gunean eta berretsi zure kontua konfiguratuta "
"dagoela; gero, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Reported Closed (422): We were unable to update your account "
"information because it appears one or more of your related accounts have "
"been closed.  Please deactivate or delete the relevant account and try "
"again."
msgstr ""
"Kontu Informatu Itxia (422): Ezin izan genuen zure kontuko informazioa "
"eguneratu, zerrendako kontu bat edo gehiago itxi dela ematen baitu. "
"Desaktibatu edo ezabatu dagokion kontua, eta saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Type Not Supported(417):The type of account we found is not "
"currently supported.  Please remove this site and add as a  manual account."
msgstr ""
"Kontu-mota ez da bateragarria(417): gaur egun aurkitzen dugun kontu-mota ez "
"da onartzen. Mesedez, hartu leku hau eta gehitu eskuz."

#. module: account_yodlee
#: model:ir.model.fields,help:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Active/Inactive on Yodlee system"
msgstr "Aktibatu/Desaktibatu Yodlee sisteman"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "An Error has occurred (code %s)"
msgstr "Errore bat gertatu da (kodea %s)"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_res_company
msgid "Companies"
msgstr "Enpresak"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Could not make sense of parameters: %s"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (402): We could not update your account "
"because your username and/or password were reported to be incorrect.  Please"
" re-verify your username and password."
msgstr ""
"Kredentzialak egiaztatu behar dira (402): Ezin dugu zure kontua eguneratu "
"zure erabiltzaile-izena eta/edo pasahitza okerrak direlako. Mesedez, "
"egiaztatu berriro zure erabiltzaile-izena eta pasahitza."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (420):The site has merged with another. "
"Please re-verify your credentials at the site and update the same."
msgstr ""
"Kredentzialak berriro agertu behar dira (420): Lekua beste batekin fusionatu"
" da. Mesedez, egiaztatu berriro zure kredentzialak gunean eta eguneratu."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s, reference code: %s"
msgstr "Errora %s, mezua: %s, erreferentzia kodea:%s"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Home Value Not Found (511): The site is unable to provide home value for "
"your property. We suggest you to delete this site."
msgstr ""
"Hasierako balioa ez da aurkitu (511): guneak ezin du etxerako balioa eman "
"zure jabetzarako. Gune hau ezabatzea gomendatzen dizugu."

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_journal
msgid "Interface for Online Account Journal"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Invalid Language Setting (421): The language setting for your site account "
"is not English. Please visit the site and change the language setting to "
"English."
msgstr ""
"Hizkuntza baliogabea konfiguratzea (421): zure gune-konturako hizkuntzaren "
"konfigurazioa ez da ingelesez. Mesedez, bisitatu gunea eta aldatu "
"hizkuntzaren konfigurazioa ingelesera."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Additional Authentication Enrollment Required (521) : Please "
"enroll in the new security authentication system, <Account Name> has "
"introduced. Ensure your account settings in <Cobrand> are updated with this "
"information."
msgstr ""
"MFA errorea: Autentifikazioko inskripzio gehigarria behar da (521): Mesedez,"
" <Account Name>sartu duzu segurtasun-autentifikazioko sistema berrian izena "
"emateko. Ziurtatu zure kontuaren konfigurazioa informazio<Cobrand> honekin "
"eguneratzen dela."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Expired (524):We're sorry, the "
"authentication information you provided has expired. Please try again."
msgstr ""
"MFA errorea: Iraungitako autentifikazio-informazioa (524): Sentitzen dugu, "
"emandako kautotze-informazioa iraungi egin da. Mesedez, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (520):We're sorry, the site "
"indicates that the additional authentication information you provided is "
"incorrect. Please try updating your account again."
msgstr ""
"MFA errorea: autentifikazio-informazio okerra (520):Sentitzen dugu, guneak "
"adierazten du emandako kautotze-informazio gehigarria ez dela zuzena. Saiatu"
" berriro kontua eguneratzen."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (523):We're sorry, the "
"authentication information you  provided is incorrect. Please try again."
msgstr ""
"MFA errorea: autentifikazio-informazio okerra (523): sentitzen dugu, "
"emandako kautotze-informazioa ez da zuzena. Mesedez, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Required (519): Your account was not "
"updated as your authentication information like security question and answer"
" was unavailable or incomplete. Please update your account settings."
msgstr ""
"MFA errorea: Eskatutako autentifikazio-informazioa (519): Zure kontua ez zen"
" eguneratu, zure kautotze-informazioa ez baitzegoen eskuragarri edo osatu "
"gabe. Mesedez, eguneratu zure kontuaren konfigurazioa."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Unavailable (518):Your account was not"
" updated as the required additional authentication information was "
"unavailable. Please try now."
msgstr ""
"MFA errorea: autentifikazio-informazioa ez dago erabilgarri (518): Zure "
"kontua ez da eguneratu, eskatutako autentifikazio gehigarriko informazioa ez"
" baitzegoen eskuragarri. Mesedez, saiatu orain."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Credential Re-Verification Required (526): We could not update "
"your account because your username/password or additional security "
"credentials are incorrect. Please try again."
msgstr ""
"MFA errorea: Eskatutako kredentzialak egiaztatzea (526): Ezin dugu zure "
"kontua eguneratu zure erabiltzaile-izena/pasahitza edo segurtasun-"
"kredentzial gehigarriak okerrak direlako. Mesedez, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Problem Updating Account (525): We could not update your account "
"for technical reasons. This type of error is usually resolved in a few days."
" We apologize for the inconvenience. Please try again later."
msgstr ""
"MFA errorea: kontua eguneratzeko arazoa (525): arrazoi teknikoengatik ezin "
"izan genuen zure kontua eguneratu. Horrelako akatsak egun gutxitan "
"konpontzen dira. Barkatu eragozpenak. Mesedez, saiatu berriro geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Request Timed Out (522) :Your request has timed out as the "
"required security information was unavailable or was not provided within the"
" expected time. Please try again."
msgstr ""
"MFAren errorea: Eskaera programatua (522): Zure eskaera amaitu da eskatutako"
" segurtasun-informazioa eskuragarri ez zegoenean edo espero zen denboran "
"eman ez denean. Mesedez, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Site Device Information Expired(509): We're sorry, we can't "
"update your account because your token is no longer valid at the site. "
"Please update your information and try again, or contact customer support."
msgstr ""
"MFAren errorea: Iraungitako guneko gailuari buruzko informazioa(509): "
"Sentitzen dugu, ezin dugu zure kontua eguneratu zure tokena ez delako "
"baliozkoa gunean. Mesedez, eguneratu zure informazioa eta saiatu berriro, "
"edo jarri harremanetan bezeroarentzako arreta-zerbitzuarekin."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Multiple User Logins(416): We attempted to update your account, but another "
"session was already established at the same time.  If you are currently "
"logged on to this account directly, please log off and try after some time"
msgstr ""
"Erabiltzaile-erregistro ugari(416): Zure kontua eguneratzen saiatzen gara, "
"baina beste saio bat ezarri zen aldi berean. Kontu honetara zuzenean "
"konektatuta bazaude, mesedez, hasi saioa eta saiatu denbora baten ondoren"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"New Login Information Required(506):We're sorry, to log in to this site, you"
" need to provide additional information. Please update your account and try "
"again."
msgstr ""
"Saioa hasteko informazio berria behar da (506): sentitzen dugu, gune honetan"
" saioa hasteko informazio gehigarria eman behar da. Mesedez, eguneratu zure "
"kontua eta saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Account Found (423): We were unable to detect an account. Please verify "
"that you account information is available at this time and If the problem "
"persists, please contact customer support at online@odoo.com for further "
"assistance."
msgstr ""
"Ez da konturik aurkitu (423): Ezin izan genuen kontu bat detektatu. Mesedez,"
" egiaztatu kontuaren informazioa une honetan eskuragarri dagoela eta Arazoak"
" bere horretan jarraitzen badu, jarri harremanetan bezeroarentzako arreta-"
"zerbitzuarekin linea@odoo.com helbidean laguntza gehiago izateko."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Payees Found(512):Your request cannot be completed as no payees were "
"found in your account."
msgstr ""
"Ez da pailazorik aurkitu(512):Zure eskaera ezin da osatu, zure kontuan ez "
"baita pailazorik aurkitu."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Online institution"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Please configure an email in the company settings."
msgstr ""
"Por favor configure un correo electrónico en la configuración de la empresa."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (406): We could not update your account because the"
" site requires you to perform some additional action. Please visit the site "
"or contact its customer support to resolve this issue. Once done, please "
"update your account credentials in case they are changed else try again."
msgstr ""
"Kontua eguneratzeko arazoa (406): Ezin dugu zure kontua eguneratu, guneak "
"ekintza osagarriren bat egitea eskatzen baitu. Mesedez, joan gunera edo "
"jarri harremanetan bezeroarentzako arreta-zerbitzuarekin arazo hori "
"konpontzeko. Hori egin ondoren, mesedez, eguneratu zure kontu-egiaztagiriak "
"beste modu batera aldatuz gero, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (517): We are sorry, there was a technical problem "
"updating your account. Please try again."
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(401):We're sorry, your request timed out. Please "
"try again."
msgstr ""
"Kontua eguneratu(401) arazoa: sentitzen dugu, zure eskaera amaitu da. "
"Mesedez, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(403):We're sorry, there was a technical problem "
"updating your account. This kind of error is usually resolved in a few days."
" Please try again later."
msgstr ""
"Arazoa Kontua eguneratzen (403): Sentitzen dugu, arazo tekniko bat izan da "
"zure kontua eguneratzen. Horrelako akatsak egun gutxitan konpontzen dira. "
"Mesedez, saiatu berriro geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(404):We're sorry, there was a technical problem "
"updating your account. Please try again later."
msgstr ""
"Arazoa Kontua eguneratzen (404): Sentitzen dugu, arazo tekniko bat izan da "
"zure kontua eguneratzen. Mesedez, saiatu berriro geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(409): We could not update your account because the "
"end site is experiencing technical difficulties."
msgstr ""
"Problema Eguneratu kontua(409): Ezin dugu zure kontua eguneratu, azken tokia"
" zailtasun teknikoak izaten ari baita."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(412): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Arazoa Eguneratu kontua(412): Ezin dugu zure kontua eguneratu lekua "
"zailtasun teknikoak izaten ari delako."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(413):We're sorry, we couldn't update your account "
"at the site because of a technical issue. This type of problem is usually "
"resolved in a few days. Please try again later."
msgstr ""
"Arazoa Eguneratu kontua(413): Sentitzen dugu, ezin genuen zure kontua lekuan"
" eguneratu arazo tekniko bat zela eta. Horrelako arazoak egun gutxitan "
"konpontzen dira. Mesedez, saiatu berriro geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(415): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Problema Eguneratu kontua(415): Ezin dugu zure kontua eguneratu lekua "
"zailtasun teknikoak izaten ari delako."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(418): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Problema Eguneratu kontua(418): Ezin dugu zure kontua eguneratu, lekua "
"zailtasun teknikoak izaten ari baita. Saiatu geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(419):We're sorry, we couldn't update your account "
"because of unexpected variations at the site. This kind of problem is "
"usually resolved in a few days. Please try again later."
msgstr ""
"Arazoa Eguneratu kontua(419):Sentitzen dugu, ezin genuen zure kontua "
"eguneratu, ustekabeko aldaketak gertatu baitira gunean. Horrelako arazoak "
"egun gutxitan konpontzen dira. Mesedez, saiatu berriro geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(425): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Problema Eguneratu kontua(425): Ezin dugu zure kontua eguneratu lekua "
"zailtasun teknikoak izaten ari delako. Saiatu geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(426): We could not update your account for "
"technical reasons. This type of error is usually resolved in a few days. We "
"apologize for the inconvenience."
msgstr ""
"Problema Eguneratu kontua(426): Ezin dugu zure kontua eguneratu arrazoi "
"teknikoengatik. Horrelako akatsak egun gutxitan konpontzen dira. Barkatu "
"eragozpenak."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(507):We're sorry, Yodlee has just started providing"
" data updates for this site, and it may take a few days to be successful as "
"we get started. Please try again later."
msgstr ""
"Kontua eguneratzeko arazoak(507):Sentitzen dugu, Yodlee gune honetarako "
"datu-eguneratzeak ematen hasi da, eta baliteke egun batzuk behar izatea hasi"
" ahala arrakasta izateko. Mesedez, saiatu berriro geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Property Record Not Found (510): The site is unable to find any property "
"information for your address. Please verify if the property address you have"
" provided is correct."
msgstr ""
"Jabetzaren erregistroa ez da aurkitu (510): Guneak ezin du bere helbiderako "
"jabetzaren informaziorik aurkitu. Egiaztatu emandako jabetza-helbidea zuzena"
" den."

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_provider__provider_type
msgid "Provider Type"
msgstr ""

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_provider
msgid "Provider for online account synchronization"
msgstr "Modulu hau lineako banku-sinkronizaziorako erabiltzen da"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-Verification Required (429): We could not update your account due to the "
"site requiring you to verify your personal information. Please log in to the"
" site and update the fields required."
msgstr ""
"Eskatutako birmatxinada (429): Ezin dugu zure kontua eguneratu, zure "
"informazio pertsonala egiaztatu behar duzun lekua dela eta. Mesedez, hasi "
"saioa gunean eta eguneratu eskatutako eremuak."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (427): We could not update your account due to the "
"site requiring you to view a new promotion. Please log in to the site and "
"click through to your account overview page to update the account.  We "
"apologize for the inconvenience."
msgstr ""
"Eskatutako berregiaztapena (427): Ezin dugu zure kontua eguneratu, sustapen "
"berri bat ikusi behar duzun lekua dela eta. Mesedez, hasi saioa gunean eta "
"egin klik zure kontuaren laburpen-orriaren bidez, kontua eguneratzeko. "
"Barkatu eragozpenak."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (428): We could not update your account due to the "
"site requiring you to accept a new Terms & Conditions. Please log in to the "
"site and read and accept the T&C."
msgstr ""
"Eskatutako berregiaztapena (428): Ezin dugu zure kontua eguneratu, "
"zehaztapen eta baldintza berriak onartzea eskatzen duen tokia dela eta. "
"Mesedez, hasi saioa gunean eta irakurri eta onartu T&C."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (433): Auto registration is not complete. "
"Please complete your registration at the end site. Once completed, please "
"complete adding this account."
msgstr ""
"Erregistroak arreta behar du (433): Erregistro automatikoa ez dago osatuta. "
"Mesedez, osatu erregistroa amaierako gunean. Osatu ondoren, mesedez, gehitu "
"kontu hau."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (434): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Erregistroak arreta behar du (434): zure erregistro automatikoa ezin izan da"
" osatu eta zuri buruzko informazio gehiago behar du. Mesedez, egiaztatu "
"berriro erregistro-informazioa prozesua osatzeko."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (435): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Erregistroak arreta behar du (435): zure erregistro automatikoa ezin izan da"
" osatu, eta informazio gehiago behar du zuri buruz. Mesedez, egiaztatu "
"berriro erregistro-informazioa prozesua osatzeko."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Request Timed Out (508): We are sorry, your request timed out due to "
"technical reasons. Please try again."
msgstr ""
"Timed Out eskaera (508): Sentitzen dugu, zure eskaera arrazoi teknikoengatik"
" amaitu da. Mesedez, saiatu berriro."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Requested Account Type Not Found (414): We could not find your requested "
"account. You may have selected a similar site under a different category by "
"accident in which case you should select the correct site."
msgstr ""
"Eskatutako kontu-mota ez da aurkitu (414): Ezin izan dugu aurkitu eskatutako"
" kontua. Istripuaren ondorioz, antzeko leku bat aukeratu duzu beste "
"kategoria batean; kasu horretan, gune zuzena hautatu behar duzu."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Server not reachable, please try again later"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Down for Maintenance(424):We were unable to update your account as the "
"site is temporarily down for maintenance. We apologize for the "
"inconvenience.  This problem is typically resolved in a few hours. Please "
"try later."
msgstr ""
"Site Down for Maintenance(424):Ezin izan genuen zure kontua eguneratu, gunea"
" aldi baterako zerbitzuz kanpo dagoelako. Barkatu eragozpenak. Arazo hori "
"ordu gutxitan konpondu ohi da. Saiatu geroago."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Available (411):The site no longer provides online services "
"to its customers.  Please delete this account."
msgstr ""
"Gunea Ez dago erabilgarri (411): Guneak ez die lineako zerbitzurik "
"eskaintzen bezeroei. Mesedez, ezabatu kontu hau."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Supported (430):This site is no longer supported for data "
"updates. Please deactivate or delete your account. We apologize for the "
"inconvenience."
msgstr ""
"Gunea Ez da bateragarria (430):Gune hau ez da bateragarria datuak "
"eguneratzeko. Mesedez, desaktibatu edo ezabatu zure kontua. Barkatu "
"eragozpenak."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Not Supported (505): We currently does not support the security system "
"used by this site. We apologize for any inconvenience. Check back "
"periodically if this situation has changed."
msgstr ""
"Leku bateraezina (505): Gaur egun ez dugu gune honek erabiltzen duen "
"segurtasun sistema babesten. Barkatu eragozpenak. Berrikusi aldizka egoera "
"hori aldatu den."

#. module: account_yodlee
#: model:ir.actions.client,name:account_yodlee.yodlee_widget
msgid "Synchronization process completed"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Timeout: the server did not reply within 30s"
msgstr "Timeout: zerbitzariak ez zuen erantzun 30 s-an"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Update Request Canceled(405):Your account was not updated because you "
"canceled the request."
msgstr ""
"Eguneratzeko eskaera bertan behera utzi da(405):Zure kontua ez da eguneratu,"
" eskaera bertan behera utzi baitzuen."

#. module: account_yodlee
#: model:ir.model.fields.selection,name:account_yodlee.selection__account_online_provider__provider_type__yodlee
msgid "Yodlee"
msgstr ""

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Yodlee Account Status"
msgstr "Yodlee kontu egoera"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_status_code
msgid "Yodlee Status Code"
msgstr "Yodlee kode egoera"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_access_token
msgid "Yodlee access token"
msgstr "Yodlee fitxa"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_login
msgid "Yodlee login"
msgstr "Yodlee login"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_password
msgid "Yodlee password"
msgstr "Yodlee pasahitza"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_access_token
msgid "access_token"
msgstr "fitxa"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "message: %s"
msgstr ""
